values.yaml

```diff
@@ -178,6 +178,9 @@ syncCatalog:
   # Set this to false to skip syncing ClusterIP services.
   syncClusterIPServices: true
 
+  # aclRequestToken used to set CONSUL_HTTP_TOKEN environment
+  aclRequestToken: null
```

Comment:
Rather than putting the token directly in to the values file, it would be more secure to reference a kubernetes secret instead. There's an example of this in PR #104 for the `enterpriseLicense` option.